data/campaigns/Dead_Water/scenarios/01_Invasion.cfg

```diff
@@ -347,12 +347,12 @@
 
         [message]
             speaker=Gwabbo
-            message= _ "Kai, please lead us against them yourself, it will hearten the folk. And — if you will permit — you reach the age now where a kai must learn battle skills."
+            message= _ "Kai, please lead us against them yourself, it will hearten the folk. And — if you will permit — you have now reached the age where a kai must learn battle skills."
```

Comment:
I think this was fine before; phrasing this in the present tense seems like a valid archaicism

Reply:
would you accept "you now reach the age"?

Reply:
I guess, but let's see what others have to say

Reply:
There's also the possibility of "you reach now the age…".

I'm not sure about the tense thing. Objectively, it seems correct to me to use the perfect present (ie "have reached"), since it's describing a state that started in the past but is still ongoing. That's why I didn't say anything about it earlier. On the other hand, it doesn't entirely sound _wrong_ to use the simple present instead. They _are_ both present tenses, so I guess that's why it still works in the old way.

Reply:
@nemaara - Any thoughts on this?

Reply:
@beetlenaut? @eric-s-raymond?

Reply:
As a native speaker, I can say that the present perfect would probably be used today. However, the simple present has a more archaic sound as cooljeanius said, so I prefer it for Wesnoth. (It's probably ESR's language.) "Now reach the age" as max-torch suggested does sound slightly better though.

Reply:
https://github.com/wesnoth/wesnoth/pull/6139/commits/dd0edc72cbd4ebba643724bb585d059ba4774e18